.github/workflows/enforceTestCoverage.yml

```diff
@@ -18,20 +18,31 @@ jobs:
       pull-requests: write
       contents: read
     env:
-      ETH_NODE_URI_MAINNET: ${{ secrets.ETH_NODE_URI_MAINNET }}
-      # keep the rest in alphabetic order
-      ETH_NODE_URI_ARBITRUM: ${{ secrets.ETH_NODE_URI_ARBITRUM }}
-      ETH_NODE_URI_BASE: ${{ secrets.ETH_NODE_URI_BASE }}
-      ETH_NODE_URI_BSC: ${{ secrets.ETH_NODE_URI_BSC }}
-      ETH_NODE_URI_GNOSIS: ${{ secrets.ETH_NODE_URI_GNOSIS }}
-      ETH_NODE_URI_GOERLI: ${{ secrets.ETH_NODE_URI_GOERLI }}
-      ETH_NODE_URI_POLYGON: ${{ secrets.ETH_NODE_URI_POLYGON }}
       MIN_TEST_COVERAGE: ${{ secrets.MIN_TEST_COVERAGE }}
     steps:
       - uses: actions/checkout@v4.1.7
         with:
           fetch-depth: 0
 
+      - name: Set up Bun
+        uses: oven-sh/setup-bun@v2
+      
+      - name: Install dev dependencies
+        run: bun install
+
+      - name: Fetch RPC endpoints from MongoDB
+        run: |
+          # Run the fetch-rpcs.ts script to update .env with RPC endpoint variables
+          bun fetch-rpcs
+          # Inject the variables from .env into the GitHub Actions environment
+          while IFS= read -r line || [[ -n "$line" ]]; do
+            if [[ "$line" != "" ]]; then
+              echo "$line" >> $GITHUB_ENV
+            fi
+          done < .env
+        env:
+          MONGODB_URI: ${{ secrets.MONGODB_URI }}
```

Comment:
Can we standardize the env var name? Here we use `MONGODB_URI` while in `.github/workflows/diamondEmergencyPause.yml` we use `MONGO_URI`

Reply:
good catch, updated